scripts/rspec/rspec.ps1

```diff
@@ -94,7 +108,11 @@ function GenerateRuleClassesVB() {
     }
 
     $FilesMap["Rule.VB.cs"] = "${RulesFolderVB}\\${ClassName}.cs"
-    $FilesMap["TestCase.VB.vb"] = "${TestCasesFolder}\\${ClassName}.vb"
+
+    if (-Not (Test-Path -Path "${TestCasesFolder}\\${className}.vb" -PathType Leaf))
```

Comment:
Is it case-insensitive?
```suggestion
    if (-Not (Test-Path -Path "${TestCasesFolder}\\${ClassName}.vb" -PathType Leaf))
```

Reply:
It is not case-sensitive, I updated it anyway so it is the same in the function.